src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -249,5 +252,35 @@ internal static MauiAppBuilder RemapForControls(this MauiAppBuilder builder)
 
 			return builder;
 		}
+
+		internal static void AddDefaultMappings()
+		{
+#if GTK
+			ContentViewHandler.Mapper.AppendToMapping(
+				"BorderColor",
+				(IContentViewHandler handler, IContentView view) =>
+				{
+					if (view is Frame frame)
+					{
+						Microsoft.Maui.Platform.ContentView contentView = (Microsoft.Maui.Platform.ContentView)handler.PlatformView;
+						contentView.SetStyleValueNode($"1px solid {frame.BorderColor.ToGdkRgba().ToString()}", contentView.CssMainNode(), "border");
+					}
+				}
+			);
+
+			ContentViewHandler.Mapper.AppendToMapping(
+				"CornerRadius",
+				(IContentViewHandler handler, IContentView view) =>
+				{
+					if (view is Frame frame)
+					{
+						Console.WriteLine($"1px solid {frame.CornerRadius:f}px");
```

Comment:
Looks like leftover from debugging session.

Reply:
oops right looks like I've missed that one. but the general idea is ok right? I mean having the mappings here instead of the app?

Reply:
Great!